src/Magick.NET/Helpers/ByteArrayWrapper.cs

```diff
@@ -3,34 +3,76 @@
 
 using System;
 using System.IO;
+#if NETSTANDARD2_1_OR_GREATER
+using System.Buffers;
+#endif
 
 namespace ImageMagick;
 
 internal sealed unsafe class ByteArrayWrapper
 {
+    #if NETSTANDARD2_1_OR_GREATER
+
+    private static readonly ArrayPool<byte> _pool = ArrayPool<byte>.Create(1024 * 1024 * 64, 1024);
+
+    private byte[] _bytes = _pool.Rent(1024 * 1024);
+
+    #else
+
     private byte[] _bytes = new byte[8192];
-    private int _offset = 0;
-    private int _length = 0;
+
+    #endif
+    private int _offset;
+
+    private int _length;
+
+    #if NETSTANDARD2_1_OR_GREATER
```

Comment:
In the rest of the project this `#if` is at the start of the line. Can you please change this.